airsonic-main/src/main/java/org/airsonic/player/service/search/IndexManager.java

```diff
@@ -62,14 +65,40 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(IndexManager.class);
 
-    // XXX 3.x -> 8.x : #1166 Holds a tentative literal. It needs to be resolved by the release.
-    private static final String INDEX_ROOT_DIR_NAME = "lucene2";
```

Comment:
Change the root directory name and fix it.
Index-related file name definitions are defined by the functions at the beginning of IndexManager.